### ISSUE TITLE
Add a clear statement about the licence and the contributor Agreement

### ISSUE THREAD

**Issue by do3cc** [plone/training#62]
As I've learned from https://github.com/plone/ansible.plone_server/pull/22#issuecomment-92301537

We don't necessarily need a CA from people to contribute to this repo.
We should clarify and state this if everybody agrees.
@pbauer

**Comment by idgserpro** [comment]
So, will the License be only a line long?

```
The Mastering Plone Training is licensed under a Creative Commons Attribution 4.0 International License.
```

**Comment by do3cc** [comment]
No, I think we should put a copy of the licence in the repo.
And we should somehow clarify if we need a contributor agreement. Imho we don't but I want Philips opinion too.

**Comment by idgserpro** [comment]
@polyester @do3cc @pbauer do you have the same opinion for [docs.plone.org](http://docs.plone.org/license.html)?

**Comment by polyester** [comment]
It's a bit complicated, but in general:

- you need a Contributor Agreement to be allowed into the "plone" organisation on github to have direct push rights. That won't change.
- for pulling in pull requests from outside contributors, that is a bit more shady. On 'configuration' and 'databases', no copyright is valid, so that can be pulled in without any doubt.
- the jury (well, german courts...) is still out on where the border lies between a trivial grammar-fix and a substantial edit that is a separate 'work' worthy of copyright protection. Yeah, lawyers...
- on longer prose which falls under the Creative Commons 4 (like training and docs.plone.org) a Contributor Agreement is ideal, but otherwise a pull request can also be seen as consent of publishing under CC4.

btw, it's customary to *not* have a copy of the license in Creative Commons-licensed websites, but to link to http://creativecommons.org/licenses/by/4.0/ (because it's clearer than the legal code)

**Comment by idgserpro** [comment]
If it's a consensus in Plone Board that " a pull request can also be seen as consent of publishing under CC4.", IMHO this should be explicitly stated in each docs/configuration repos contributing guidelines, or even in [Contributor’s Agreement for Plone Explained](https://plone.org/foundation/contributors-agreement) itself.

I'm telling this because, as a corporate account, we can't change repos that need a Plone Contributor's Agreement. It would be nice to have this change explicity stated and possible to be audited by contractors instead of a reference of this subject in unrelated github issues.

I know this is kind of a "grey area", we're just giving our 2 cents.

**Comment by stevepiercy** [comment]
This has been fixed in #526.
